src/Classes/Options/Interactivity/Modes/Grab.ts

```diff
@@ -0,0 +1,30 @@
+import {IGrab} from "../../../../Interfaces/Options/Interactivity/Modes/IGrab";
+import {IGrabLineLinked} from "../../../../Interfaces/Options/Interactivity/Modes/IGrabLineLinked";
+import {GrabLineLinked} from "./GrabLineLinked";
+
+export class Grab implements IGrab {
+    /**
+     *
+     * @deprecated this property is obsolete, please use the new lineLinked
+     */
+    public get line_linked(): IGrabLineLinked {
+        return this.lineLinked;
+    }
+
+    /**
+     *
+     * @deprecated this property is obsolete, please use the new lineLinked
+     * @param value
+     */
+    public set line_linked(value) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected parameter: 'value' to have a typedef](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5136254)